src/20240617_AaveV3Avalanche_UpdatePoRExecutorV3Robot/UpdatePoRExecutorV3Robot_20240617.s.sol

```diff
@@ -0,0 +1,73 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {GovV3Helpers, IPayloadsControllerCore, PayloadsControllerUtils} from 'aave-helpers/GovV3Helpers.sol';
+import {GovernanceV3Ethereum} from 'aave-address-book/GovernanceV3Ethereum.sol';
+import {EthereumScript, AvalancheScript} from 'aave-helpers/ScriptUtils.sol';
+import {AaveV3Avalanche_UpdatePoRExecutorV3RobotCancel_20240617} from './AaveV3Avalanche_UpdatePoRExecutorV3RobotCancel_20240617.sol';
+import {AaveV3Avalanche_UpdatePoRExecutorV3RobotRegister_20240617} from './AaveV3Avalanche_UpdatePoRExecutorV3RobotRegister_20240617.sol';
+
+/**
+ * @dev Deploy Avalanche
+ * deploy-command: make deploy-ledger contract=src/20240617_AaveV3Avalanche_UpdatePoRExecutorV3Robot/UpdatePoRExecutorV3Robot_20240617.s.sol:DeployAvalanche chain=avalanche
+ * verify-command: FOUNDRY_PROFILE=avalanche npx catapulta-verify -b broadcast/UpdatePoRExecutorV3Robot_20240617.s.sol/43114/run-latest.json
+ */
+contract DeployAvalanche is AvalancheScript {
+  function run() external broadcast {
+    // deploy payloads
+    address payload0 = GovV3Helpers.deployDeterministic(
+      type(AaveV3Avalanche_UpdatePoRExecutorV3RobotCancel_20240617).creationCode
+    );
+
+    address payload1 = GovV3Helpers.deployDeterministic(
+      type(AaveV3Avalanche_UpdatePoRExecutorV3RobotRegister_20240617).creationCode
+    );
+
+    // compose action
+    IPayloadsControllerCore.ExecutionAction[]
+      memory actions = new IPayloadsControllerCore.ExecutionAction[](2);
+    actions[0] = GovV3Helpers.buildAction(payload0);
```

Comment:
@rustboyar We should create 2 payloads instead of 2 actions for a single payload (and change the proposal creation script also), as with the current approach it will try to execute payload0 and payload1 in the same tx.